src/promptflow/promptflow/_sdk/data/visualize.j2

```diff
@@ -10,6 +10,6 @@
     <script>
       window.bulk_test_details_data = {{ data }}
     </script>
-    <script src="{{ js_path }}"></script>
+    <script src="https://sdk-bulk-test-endpoint.azureedge.net/bulk-test-details/view/0.0.35/bulkTestDetails.min.js"></script>
```

Comment:
I understand as this CDN link will not be updated anymore, so we don't need the suffix `?version=1` anymore.

+ @RealKai42 to help confirm